runtime_k3d/src/runtime/runtime.py

```diff
@@ -43,7 +43,7 @@ def is_runtime_installed(log_output: TextIOWrapper | int = subprocess.DEVNULL) -
         if key == "STATUS" and value == "deployed":
             return True
 
-    log_output.write(process.stdout)
+    log_output.write(process.stdout)  # type: ignore
```

Comment:
I don't like the "code pollution" with linter comments. Also, there are too many places where this needs being added.
Is there no option to either fix it or comment it out "globally"?

Reply:
I totally agree and would like to really fix this, but thought that this is out of scope of this PR. 

There are different solutions how to fix it, one would be to add a log function to the velocitas-lib, another one would be to simply force a valid logfile everywhere.

TLDR: needs more discussion

Reply:
fixed